### DISCUSSION TITLE
Reduce Initial Installation Background Noise

### DISCUSSION THREAD

**Discussion by loganterfehr** [Security-Onion-Solutions/securityonion#12175] [Unsupported Versions]
I am curious about how to reduce a lot of the noise that just comes from having a manager, search, and sensor node. There are 1000's of logs every couple minutes. How could I go about creating a rule(s) to quiet these. I only have those three nodes installed, is there really that much traffic between them when nothing else is added? Thanks ahead of time.

**Comment by reyesj2** [comment]
Can you give an example of the noise you're referring to?

**Comment by loganterfehr** [comment]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/122332360/da3f9d20-d91f-4ba4-a4c9-67e07a848052)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/122332360/f73fe445-beb8-4561-8275-f52be4a73fda)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/122332360/b74533dd-aa37-4d5b-b18e-7f386ad3ecfd)
(logs similar to these)

There are like 300x the amount of logs as a regular host. Is this typical?

**Reply by reyesj2** [reply]
If you use SOC and go to Dashboards then click the drop down and select the "Host Overview" dashboard you can get an overview of what logs are being collected
